app/plugins/router.js

```diff
@@ -29,10 +29,13 @@ const routes = [].concat(
   require('../routes/vet/practice'),
   require('../routes/vet/rcvs'),
   require('../routes/vet/reference'),
+  require('../routes/vet/visit-date'),
```

Comment:
Can the routes be added alphabetically please, it makes then easier to read through.